mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'two-third-third.html' %}
+
+{% load i18n static %}
+
+{% block twothird %}
+    <h2><span class="theme read">{{ document.get_type_display }}</span> {{ document }}</h2>
+    {% if document.type == document.IMAGE %}
+        <img src="{{ document.original.url }}" />
+    {% elif document.type == document.VIDEO %}
+        <video controls width="100%">
+            <source src="{{ document.original.url }}">
+        </video>
+    {% else %}
+        <div>
+            <a href="{{ document.original.url }}">
+                <img src="{% static 'mediacenter/document.svg' %}" /><br />{% trans 'Download document' %}
+            </a>
+        </div>
+    {% endif %}
+    <div class="text">{{ document.summary }}</div>
+{% endblock twothird %}
+{% block third %}
+    {% if user.is_staff %}
+        <ul class="card tinted admin">
+            <li><a href="{% url 'mediacenter:document_update' pk=document.pk %}">{% trans 'Edit' %}</a></li>
```

Comment:
instead to rely on long urls, I prefer defining quick methods in the model (`get_update_url`...)... More maintainable and readable, for me.